scipy/integrate/vode.py

```diff
@@ -2,10 +2,7 @@
 
 from scipy._lib.deprecation import _sub_module_deprecation
 
-__all__ = [  # noqa: F822
-    'dvode',
-    'zvode'
-]
+__all__ = []
```

Comment:
In the meantime, one thing to think about is these empty modules. On the one hand, we said 2.0 but if they are empty...

Reply:
Actually, upon reflection probably worth keeping as it provides a more helpful warning